README.md

```diff
@@ -21,7 +21,7 @@ The ChartboostExample project includes two targets:
 
 - `ChartboostExample`: Manual dependency management. 
 
- It requires the `Chartboost.framework` to be dropped into the root directory (where this readme is).
+It requires the `Chartboost.framework`, is already in the root directory (where this readme is) and linked to the Xcode project.
```

Comment:
A small grammar correction:
"It requires the Chartboost.frameworks, **which is** already in..."